package.json

```diff
@@ -21,18 +21,18 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "babel-cli": "6.5.1",
+    "babel-cli": "^6.11.4",
     "babel-core": "6.5.1",
-    "babel-plugin-lodash": "2.0.1",
-    "babel-preset-es2015": "6.5.0",
+    "babel-plugin-lodash": "^3.2.8",
+    "babel-preset-es2015": "^6.13.2",
     "chai": "3.2.0",
     "documentation": "3.0.4",
     "eslint": "1.10.2",
     "mocha": "2.4.5",
     "webpack": "1.12.13"
   },
   "dependencies": {
-    "lodash": "4.9.0"
+    "lodash": "^4.9.0"
```

Comment:
Can you revert these updates in package.json? I think it's really important to pin dependencies especially when working with fast moving projects like babel, and upgrades to this kind of tooling should go in a separate PR, if they are required.